docs/rules/prefer-string-slice.md

```diff
@@ -0,0 +1,16 @@
+# Prefer the use of `String.slice` instead of `String.substr` or `String.substring`
+
+Prefer a convention of using `String.slice` instead of `String.substr` or `String.substring`. `String.slice()` has clearer behavior and has a counterpart with arrays. It is also better to be consistent. Anything that can be done with `String.substr()` or `String.substring()` can be done with `String.slice()`.
```

Comment:
Linkify these methods to their MDN page.

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/slice